src/middlewares/errorHandler/errorHandler.middleware.ts

```diff
@@ -0,0 +1,12 @@
+import { Request, Response, NextFunction, ErrorRequestHandler } from 'express';
+
+const ErrorHandler: ErrorRequestHandler = (
+  err: Error,
```

Comment:
Is Error a standard typescript type or should that also be imported from 'express'?

Reply:
I think it's a Typescript standard type.